content/c-sharp/concepts/math-functions/terms/sin/sin.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: '.Sin()'
+Description: 'Returns the sine of a given angle.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Numbers'
+  - 'Arithmetic'
+  - 'Functions' 
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`Math.Sin()`** class method returns the sine of a given angle.
+
+## Syntax
+
+```pseudo
+Math.Sin(angle);
+```
+
+The `Math.Sin()` method takes only one `double` type parameter - `angle` - an angle in radians. The `Math.Sin()` method returns the sine of the `angle` as a `double` value  or `NaN` (not a number) if the `angle` equals to:
+
+- `NaN`,
+- `NegativeInfinity`, or
+- `PositiveInfinity`.
+
+## Example
+
+The following example first converts `30` degrees to radians and then uses `Math.Sin()` method to return the sine of that angle. Finally, the `Console.WriteLine()` function prints the result to the console:
+
+```cs
+using System;
+
+public class Example {
+  public static void Main(string[] args) {
+    double degrees = 30;
+    double radians = degrees * Math.PI/180;
+
+    double sine = Math.Sin(radians);
+
+    Console.WriteLine("The sine of " + degrees + " degrees is: " + sine);
+  }
+}
+```
+
+The example will result in the following output:
+
+```shell
+The sine of 30 degrees is: 0.5
```

Comment:
I'm getting .49999... Perhaps there should be a rounding method included.

Reply:
I get 0.5. It should be 0.5. 
![sine](https://user-images.githubusercontent.com/81328857/225271097-e7699b82-eace-4633-bb10-3bd7eea6af97.png)
I am trying to figure out what the issue is. Even if I use Math.Round(), I will get 0.5 and not 0.49999. Do you know what the issue could be?

Reply:
I added a note that explains the difference in the results.